internal/scan/scan.go

```diff
@@ -14,17 +14,28 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/freeverseio/laos-universal-node/internal/platform/blockchain/contract"
+	"github.com/freeverseio/laos-universal-node/internal/platform/model"
 )
 
 var (
-	eventTransferName              = "Transfer"
-	eventApprovalName              = "Approval"
-	eventApprovalForAllName        = "ApprovalForAll"
-	eventNewERC721Universal        = "NewERC721Universal"
-	eventTransferSigHash           = generateEventSignatureHash(eventTransferName, "address", "address", "uint256")
-	eventApprovalSigHash           = generateEventSignatureHash(eventApprovalName, "address", "address", "uint256")
-	eventApprovalForAllSigHash     = generateEventSignatureHash(eventApprovalForAllName, "address", "address", "bool")
-	eventNewERC721UniversalSigHash = generateEventSignatureHash(eventNewERC721Universal, "address", "string")
+	eventTransferName                  = "Transfer"
+	eventApprovalName                  = "Approval"
+	eventApprovalForAllName            = "ApprovalForAll"
+	eventNewERC721Universal            = "NewERC721Universal"
+	eventNewCollection                 = "NewCollection"
+	eventMintedWithExternalURI         = "MintedWithExternalURI"
+	eventEvolvedWithExternalURI        = "EvolvedWithExternalURI"
+	eventTransferSigHash               = generateEventSignatureHash(eventTransferName, "address", "address", "uint256")
+	eventApprovalSigHash               = generateEventSignatureHash(eventApprovalName, "address", "address", "uint256")
+	eventApprovalForAllSigHash         = generateEventSignatureHash(eventApprovalForAllName, "address", "address", "bool")
+	eventNewERC721UniversalSigHash     = generateEventSignatureHash(eventNewERC721Universal, "address", "string")
+	eventNewCollectionSigHash          = generateEventSignatureHash(eventNewCollection, "uint64", "address")
+	eventMintedWithExternalURISigHash  = generateEventSignatureHash(eventMintedWithExternalURI, "uint64", "uint96", "address", "string", "uint256")
+	eventEvolvedWithExternalURISgiHash = generateEventSignatureHash(eventEvolvedWithExternalURI, "uint64", "uint256", "string")
+	EventNewCollectionSigHash          = generateEventSignatureHash(eventNewCollection, "uint64", "address")
+	EventMintedWithExternalURISigHash  = generateEventSignatureHash(eventMintedWithExternalURI, "uint64", "uint96", "address", "string", "uint256")
+	EventEvolvedWithExternalURISgiHash = generateEventSignatureHash(eventEvolvedWithExternalURI, "uint64", "uint256", "string")
```

Comment:
Make it unexported

Reply:
Unused, they must have been the remains of solving merge conflicts. Thanks